index.html

```diff
@@ -15,15 +22,24 @@
 					<a href="#" class="logo">
 						<img src="./images/Welcome.svg" alt="Welcome" />
 					</a>
-					<a href="#" class="menu">
-						<img src="./images/menu.svg" alt="Drop down menu" />
-					</a>
+					<ul class="mobile-icons">
+						<li><i class="fa fa-bars fa-xl hamburger"></i></li>
+						<li><i class="fa fa-xmark fa-solid fa-xl close"></i></li>
```

Comment:
- [ ] Kindly change color of the close icon to the color used in Figma design, check screen shot below